CWE-94-Python_darwin/nccgroup_TriforceLinuxSyscallFuzzer/gen.py

```
@@ -0,0 +1,158 @@
+        #print repr(self), 'add', x.encode('hex')
+        self.buf.append(x)
+    def pack(self, fmt, *args) :
+        x = struct.pack(fmt, *args)
+        self.add(x)
+    def __str__(self) :
+        return ''.join(self.buf)
+
+class Num(object) :
+    def __init__(self, v) :
+        self.v = v
+    def mkArg(self, buf, xtra) :
```

Ruff (0.13.2)

Unused method argument: `xtra`

(ARG002)